lib/marc_cleanup/variable_fields.rb

```diff
@@ -277,18 +233,21 @@ def multiple_no_040b?(record)
     b040 = f040.subfields.select { |subfield| subfield.code == 'b' }
     return true if b040.size != 1
 
-    b040 = b040.first.value
-    b040.gsub!(/[ ]/, '')
+    b040 = b040.first.value.dup
+    b040.gsub!(/\s/, '')
     b040 == ''
```

Comment:
Is this checking if the first subfield b is empty (or only whitespace)?  I wonder if something like this would work to avoid changing the value of `b040` (and `dup`) .
```suggestion
    b040.match? /^\s*$/
```

Reply:
Yes, it is checking if the first subfield b is empty or only whitespace.

My original logic was to remove all space characters, then see if there are any other characters left. But I understand now that I don't need to trim the whitespace to be able to determine if the value is only composed of space characters (or is empty).

Thank you, this makes it clear what it's going to do. Pushing a change now.